maven/src/org/netbeans/modules/maven/execute/AbstractOutputHandler.java

```diff
@@ -66,7 +66,7 @@
     private IndexingBridge.Lock protectedMode; // #211005
     private final Object protectedModeLock = new Object();
     private RequestProcessor.Task sleepTask;
-    private static final int SLEEP_DELAY = 5000;
+    private static final int SLEEP_DELAY = Integer.getInteger(AbstractOutputHandler.class.getName() + ".SLEEP_DELAY", 15000); // #270005
```

Comment:
This property shall be documented in `arch.xml` document - `<api category="systemproperty" ..../>` - at least that used to be our habit before switching to Apache.